modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultGenerator.java

```diff
@@ -264,7 +264,8 @@ public Reader getTemplate(String name) {
                     operation.put("basePath", basePath);
                     operation.put("basePathWithoutHost", basePathWithoutHost);
                     operation.put("contextPath", contextPath);
-                    operation.put("baseName", tag);
+                   // operation.put("baseName", tag); FIXME ask @hiveship if this modification cause a problem
```

Comment:
@hiveship can you elaborate on why you replaced baseName with `basePathWithoutHost`?

`basePathWithoutHost` is the same as `contextPath` and I'll file a PR to replace `basePathWithoutHost` with `contextPath`